### DISCUSSION TITLE
希望NumberUtil里的isNumber能支持只校验十进制数字，这样更符合直觉。

### DISCUSSION THREAD

**Discussion by rain-fall-cpu** [looly/hutool#3405] [Ideas]
想通过hutool判断用户输入的是否为十进制数字，于是看hutool文档发现有isNumber的方法，但是hutool的这个方法覆盖各种进制。
比如用户输入4213d，我的需求是判断其为错误输入，但是hutool会认为它是一个数字。
实际使用场景，也会有这种只校验十进制的场景，如果能增加一个isNumber方法，能让用户指定只校验那些进制，就会比较好。

**Comment by looly** [comment]
也提供了`isisInteger` `isLong` `isDouble`等方法，考虑使用。
